src/components/Content/ContentHeader/ExperimentButtons/Container.js

```diff
@@ -64,18 +74,25 @@ const ExperimentButtonsContainer = ({
   });
 
   // HOOKS
-  // did mount hook
+  // did mount hooks
   useEffect(() => {
     const polling = setInterval(
-      () => handleFetchExperimentDeployStatus(projectId, experimentId),
+      () => handleFetchDeploymentStatus(projectId, experimentId),
       5000
     );
     return () => clearInterval(polling);
   });
 
+  useEffect(() => {
+    if (experimentId) {
+      handleFetchOperators(projectId, experimentId);
+      experiment(experimentId);
+    }
+  }, [projectId, experimentId, handleFetchOperators]);
```

Comment:
Line 91:6:  React Hook useEffect has a missing dependency: 'experiment'. Either include it or remove the dependency array. If 'experiment' changes too often, find the parent component that defines it and wrap that definition in useCallback  react-hooks/exhaustive-deps